1. Desktop app calls Web API/AppCreationScripts/Configure.ps1

```diff
@@ -283,7 +283,7 @@ Function ConfigureApplications
    Write-Host "Creating the AAD application (TodoListClient (active-directory-dotnet-native-aspnetcore-v2))"
    # create the application 
    $clientAadApplication = New-AzureADApplication -DisplayName "TodoListClient (active-directory-dotnet-native-aspnetcore-v2)" `
-                                                  -ReplyUrls "https://login.microsoftonline.com/common/oauth2/nativeclient" `
+                                                  -ReplyUrls "http://localhost", "https://login.microsoftonline.com/common/oauth2/nativeclient" `
```

Comment:
We also need to change the apps.json and sample.json
(so that the quickstart does not get broken)

Reply:
Well the WPF app.config does not have the redirect URI as a param, so I think it should be fine.